Source/Diagnostics/ParticleDiag/ParticleDiag.cpp

```diff
@@ -51,13 +51,10 @@ ParticleDiag::ParticleDiag(
             }
         }
     }
-
-#ifdef WARPX_DIM_RZ
-    // Always write out theta, whether or not it's requested,
-    // to be consistent with always writing out r and z.
-    // TODO: openPMD does a reconstruction to Cartesian, so we can now skip force-writing this
-    m_plot_flags[pc->getParticleComps().at("theta")] = 1;
-#endif
```

Comment:
My impression is that `theta` is not useful, since the particles are dumped in Cartesian coordinates anyway. In fact, that this flag is not doing anything, since `theta` is actually not written anymore as of:
https://github.com/ECP-WarpX/WarpX/pull/4686

Reply:
Hm, it seems that this is still needed when writing `plotfiles`.